templates/aws-standalone-cp/values.yaml

```diff
@@ -44,3 +44,7 @@ worker:
 # K0s parameters
 k0s:
   version: v1.30.2+k0s.0
+
+# Optionally install apps defined under 
+# templates/apps into target cluster
+installApps: false
```

Comment:
can we make this by default true? I think its just easier to demo.
Also let's rename `installApps` to `installBeachHeadServices` which is the term we want to use for these type of applications

Reply:
@wahabmk Oh also, can we rename the folder `templates/apps` to `templates/beachheadservices`